R/inbo_rapport.R

```diff
@@ -69,15 +74,18 @@ inbo_rapport <- function(
       style,
       Flanders = c(
         pandoc_variable_arg("style", "flanders_report"),
-        pandoc_variable_arg("mylanguage", "french,dutch,english")
+        pandoc_variable_arg("babel", babel),
+        pandoc_variable_arg("lang", "en")
```

Comment:
should `lang:en` not be made dynamical for the Flanders style, i.e. by using the language code that matches the last element of `babel`?

Reply:
We could do that. But what happens with the CSL when you provide something different than `nl` or `en`?

Reply:
CSL will change all [terms](https://docs.citationstyles.org/en/stable/specification.html#appendix-ii-terms) to the defaults for the chosen language (the defaults per language are in [this](https://github.com/citation-style-language/locales) repo). It is only when we want to override certain defaults, that it has been made explicit in the INBO CSL file - up to now only for `en` and `nl` and only for a few terms. We can expand that as we like, but it will not stop CSL from taking defaults for other languages and terms.